spec/controllers/users_controller_spec.rb

```diff
@@ -22,6 +22,19 @@
         expect(response).to redirect_to(new_user_session_path)
       end
     end
+
+    context 'when admin is logged and want to generate csv file' do
+      it 'generates csv file with user data' do
+        admin = create(:user, admin: true, email: 'example-admin-mail@mail.com')
+        create(:user, email: 'example-user-mail@mail.com')
+        sign_in admin
+
+        get :index, format: :csv
+
+        expect(response.body).to include('example-user-mail@mail.com')
```

Comment:
It's not very specific, and can easily lead to tests passing on a broken app. If your response will be just an HTML index, it will include those mails too.